src/Simulation/Algorithms/COE/COE.hpp

```
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <cmath>
+#include <limits>
+#include <numbers>
+
+#include <Core/Data/Physics.hpp>
+
+
+namespace COE {
+    /* Classical orbital elements */
+    struct Elements {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: p, a, e, incl, omega, argp, nu, m, arglat, truelon, lonper

(cppcoreguidelines-pro-type-member-init)